examples/basic.html

```diff
@@ -58,15 +58,19 @@ <h2>
         <dfn>Foo</dfn> interface
       </h2>
       <pre class="idl">
-      [Exposed=Window, Constructor]
+      [Exposed=Window]
       interface Foo {
+        constructor();
         attribute DOMString bar;
         void doTheFoo();
       };
       </pre>
       <p>
         The <a>Foo</a> interface is nice. Lets you do stuff.
       </p>
+      <p>
+        The <dfn>Foo.constructor()</dfn> creates a Foo instance.
```

Comment:
We still want to be able to define it as `<dfn>constructor()</dfn>` like a regular operation.

Reply:
That should work too. Will add a relevant test.